app/views/home/chat.html.erb

```diff
@@ -42,6 +42,12 @@
   Lurking in the channel is a bot creatively called <a href="https://github.com/lobsters/lobstersbot">lobstersbot</a>.
   It announces new stories submitted to the site and provides other useful features like showing the page title of URLs mentioned in the channel.
   </p>
+    
```

Comment:
Please move this paragraph up under the `<h1>`, it's the most important thing on the page.